posthog/hogql/database/schema/test/test_channel_type.py

```diff
@@ -602,6 +602,16 @@ def test_zendesk_ticket_14945(self):
                 }
             ),
         )
+        self.assertEqual(
```

Comment:
total nitpick that you get better output on failure from `assert x == y` 🙈

Reply:
Ah yes I did know that! Might as well change it